assets/bower_components/three-js/three-js.html

```diff
@@ -54,7 +46,8 @@
           });
           Polymer.dom(this.root).appendChild(renderer.domElement);
           this.renderer = renderer;
-          renderer.setSize(this.w, this.h);
+          var parent = Polymer.dom(this).parentNode;
```

Comment:
This will break in places where a outerdiv isn't specified right? Let's put "1000" and "562" as a default setting incase it isn't encased in a `div`?